### PR TITLE
Add rake task to create popular links and links to homepage

### PR DESCRIPTION
**Pull request by syed-ali-tw**
Create popular links and links to homepage
this task performs two actions
1. Create the very first popular links record in publisher db
2. Calls publishing-api patch links endpoint to link homepage and popular links content-ids

[Trello](https://trello.com/c/dTSlgdbI/302-create-a-rake-task-to-bootstrap-popular-links)

### CODE REVIEW

File: lib/tasks/create_popular_links_and_link_to_homepage.rake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+desc "Create popular links and links to homepage"
+task create_popular_links_and_link_to_homepage: [:environment] do
+  homepage_content_id = "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a".freeze
+  popular_links = PopularLinksEdition.new(title: "Homepage popular links", link_items: [{ url: "url1.com", title: "title1" }, { url: "url2.com", title: "title2" }, { url: "url3.com", title: "title3" }, { url: "url4.com", title: "title4" }, { url: "url5.com", title: "title5" }, { url: "url6.com", title: "title6" }])
```
Comment (mtaylorgds) [inline lib/tasks/create_popular_links_and_link_to_homepage.rake:4]: Is it worth populating with the current popular link values?

Comment (mtaylorgds) [inline lib/tasks/create_popular_links_and_link_to_homepage.rake:4]: Could we maybe reformat this code to be a little easier to read?

**Review by mtaylorgds — COMMENTED** [review]
Couple of minor comments; could you also add the link to the Trello card in the PR description.

File: lib/tasks/create_popular_links_and_link_to_homepage.rake (hunk 1 above)
Comment (syed-ali-tw) [reply lib/tasks/create_popular_links_and_link_to_homepage.rake:4]: I am not sure about this, the values can change between when the code is merged and when the rake task is run.

**Review by mtaylorgds — APPROVED** [review]
LGTM; Before we run the rake task for real, we should get clarification on whether it should have the real, current, popular links data.

File: lib/tasks/create_popular_links_and_link_to_homepage.rake (hunk 1 above)
Comment (mtaylorgds) [reply lib/tasks/create_popular_links_and_link_to_homepage.rake]: The team has decided that we will leave as these defaults now, and we can then guide a content designer through the process of updating to use the current values that are on the homepage.

**Comment by syed-ali-tw** [comment]
> Before we run the rake task for real, we should get clarification on whether it should have the real, current, popular links data.

Given, we discussed during standup that we dont necessarily want to populate current, popular links data yet and given, it wont be publish with rake task.
assuming we are happy to merge this.

**Pull request merged**